jekyll/_cci2/concepts.md

```diff
@@ -89,54 +89,89 @@ Jobs are the building blocks of your config. Jobs are collections of [steps](#st
+   # and docker-compose 1.14.0, follow CircleCI Discuss Announcements
+   # for new image releases.
+     image: ubuntu-1604:201903-01
 #...       
-   build3:
-     macos: # Specifies a macOS virtual machine with Xcode version 11.3
-       xcode: "11.3.0"
+ build3:
+   macos: # Specifies a macOS virtual machine with Xcode version 11.3
+     xcode: "11.3.0"
 # ...          
- ```
+```
+
+{:.tab.executors.Server}
+```yaml
+version: 2.0
+
+jobs:
+ build1: # job name
+   docker: # Specifies the primary container image,
+     - image: buildpack-deps:trusty
+     - image: postgres:9.4.1 # Specifies the database image
+      # for the secondary or service container run in a common
+      # network where ports exposed on the primary container are
+      # available on localhost.
+       environment: # Specifies the POSTGRES_USER authentication
+        # environment variable, see circleci.com/docs/2.0/env-vars/
+        # for instructions about using environment variables.
+         POSTGRES_USER: root
+#...
+ build2:
+   machine: # Specifies a machine image that uses
+   # an Ubuntu version 14.04 image with Docker 17.06.1-ce
+   # and docker-compose 1.14.0, follow CircleCI Discuss Announcements
+   # for new image releases.
+     image: ubuntu-1604:201903-01
+#...       
+ build3:
+   macos: # Specifies a macOS virtual machine with Xcode version 11.3
+     xcode: "11.3.0"
+# ...          
+```
+
+
 
 The Primary Container is defined by the first image listed in [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file. This is where commands are executed. The Docker executor spins up a container with a Docker image. The machine executor spins up a complete Ubuntu virtual machine image. See [Choosing an Executor Type]({{ site.baseurl }}/2.0/executor-types/) document for a comparison table and considerations. Subsequent images can be added to spin up Secondary/Service Containers.
 
 When using the docker executor and running docker commands, the `setup_remote_docker` key can be used to spin up another docker container in which to run these commands, for added security. For more information see the [Running Docker Commands]({{ site.baseurl }}/2.0/building-docker-images/#accessing-the-remote-docker-environment) guide.
 
 #### Steps
-{:.no_toc}
 
 Steps are actions that need to be taken to complete your job. Steps are usually a collection of executable commands. For example, the [`checkout`]({{ site.baseurl }}//2.0/configuration-reference/#checkout) step, which is a _built-in_ step available across all CircleCI projects, checks out the source code for a job over SSH. Then, the `run` step allows you to run custom commands, such as executing the command `make test`  using a non-login shell by default. Commands can also be defined [outside the job declaration]({{ site.baseurl }}/2.0/configuration-reference/#commands-requires-version-21), making them reusable across your config.
 
 ```yaml
 #...
```

Comment:
Do we need a version number here?

Reply:
As it's an extract of the config we don't need the version